recipes/default.rb

```diff
@@ -37,6 +37,10 @@
   system  true
 end
 
+user_ulimit node["kafka"]["user"] do
+  filehandle_limit 'unlimited'
```

Comment:
I think you just need to do it in multiples of 1024.

I pulled the 'unlimited' from the chef-ulimit docs, but that was also for a
different system limit.

Sent from my pocket watch

On Mar 18, 2015, at 1:42 PM, Blake Imsland <notifications@github.com> wrote:

In recipes/default.rb
<https://github.com/apptentive-engineering/chef-kafka/pull/1#discussion_r26705097>
:

> @@ -37,6 +37,10 @@
>    system  true
>  end
>
> +user_ulimit node["kafka"]["user"] do
> +  filehandle_limit 'unlimited'

IIRC "unlimited" doesn't work properly for open file descriptor limits,
I've seen various docs recommend "100000" or "65536".

—
Reply to this email directly or view it on GitHub
<https://github.com/apptentive-engineering/chef-kafka/pull/1/files#r26705097>
.